components/Card.js

```diff
@@ -0,0 +1,63 @@
+import initialCards from "@/lib/db";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function Card({}) {
```

Comment:
this component displays a List of cards. please rename it accordingly. At the moment you might think that it renders only one card.

Reply:
I think GridOfCards is good, or?